Makefile.am

```diff
@@ -227,7 +227,7 @@ ci:
 	fi
 
 container-ci:
-	$(CONTAINER_ENGINE) run $(CONTAINER_TEST_ARGS) $(CI_TASKS_TEST_ARGS) $(CI_TASKS_NAME):$(CI_TASKS_TAG)
+	$(CONTAINER_ENGINE) run $(CONTAINER_TEST_ARGS) $(CI_TASKS_NAME):$(CI_TASKS_TAG) $(CI_TASKS_TEST_ARGS)
```

Comment:
I see we need a trailing variable to fix the "run a single test" use case (I didn't think of that in #2923 as I never use that -- iterating in an interactive shell is just so much nicer). We could have a `$(CI_TASKS_CMD)` for that, reflecting the container's `CMD`?